# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: toolkit/library/rust/shared/lib.rs
# Commit: 63b555e1f503
# Full Hash: 63b555e1f503db91a54604fd8cef88dfab7c0c7a
# Author: Dana Keeler <dkeeler@mozilla.com>
# Date: 2023-06-08 09:27:30
# Regressor Bug: 1834862
# File Overlap Count: 1
# Description:
#   Bug 1834862 - implement nsICryptoHash in rust r=jschanck,supply-chain-reviewers
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D179727
# ==============================================================================

diff -r 6337ce701771 -r 63b555e1f503 toolkit/library/rust/shared/lib.rs
--- a/toolkit/library/rust/shared/lib.rs	Wed Jun 07 22:44:15 2023 +0000
+++ b/toolkit/library/rust/shared/lib.rs	Wed Jun 07 23:03:21 2023 +0000
@@ -16,6 +16,7 @@
 extern crate bookmark_sync;
 extern crate cascade_bloom_filter;
 extern crate cert_storage;
+extern crate crypto_hash;
 extern crate chardetng_c;
 extern crate cosec;
 #[cfg(feature = "cubeb_coreaudio_rust")]
